Gemfile

```diff
@@ -51,5 +52,6 @@ end
 group :test do
   gem "capybara"
   gem "selenium-webdriver"
+  gem "shoulda"
```

Comment:
I'm a little confused on why we need this. It seems to just be testing Rails itself? I don't think we need to separately verify there are certain associations, since they're being used in tests anyway.

Reply:
Ah, it's a requirement for testing `active_storage_validations`. Is there a benefit to testing that on our end? `shoulda` seems to move the DSL in the direction of `rspec`, which could get murky.

Reply:
Notes from today's one-on-one:
- We'll remove the validation tests, but keep the `shoulda` gem as it's used by `active_storage_validations`

Reply:
[`304ffde` (#53)](https://github.com/hackclub/hackathons-backend/pull/53/commits/304ffde978fb3f35095fc7a392d244a8f9ae8359)